docs/extras/save_management.md

```diff
@@ -1,19 +1,19 @@
 # Save Management
 
-For save management,  [Edizon](https://github.com/WerWolv/EdiZon/releases) is recommended. It can be used to back up and restore game saves to your SD card. It also has the ability to edit saves for extra items etc. It also has support for Atmosphere's cheat engine, providing an easy way to download new cheats, as well as toggle them on or off.
+For save management, [JKSV](https://github.com/J-D-K/JKSV) is recommended. It can be used to back up and restore game saves to your SD card. 
 
-[Checkpoint](https://github.com/flagbrew/checkpoint/releases) is also a save manager. It can be used to back up and restore game saves to your SD card. It also has the ability to share save data over FTP and WiFi.
+[Checkpoint](https://github.com/SciresM/Checkpoint/releases/tag/ams-19-compat) is also a save manager. It can be used to back up and restore game saves to your SD card. It also has the ability to share save data over FTP and WiFi.
```

Comment:
I do not think we should just wildly link scires' release on our docs page. Maybe mention that the current release is broken, and that there's a semi-fixed build [here]?